### PR TITLE
add AI usage docs for spiderpool

### PR DESCRIPTION
**Pull request by ty-dc**
See preview:
https://deploy-preview-5386--daocloud-docs.netlify.app/network/modules/spiderpool/install/ai_cluster_with_macvlan

### CODE REVIEW

**Comment by windsonsea** [comment]
If ready, pls remove the label `wip-do-not-merge`.

File: docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+    1.1. 进入 `Helm 应用` -> `Helm 模板`，找到并点击 ofed-driver
+
+    ![ofed-driver-install1](../../../images/ofed-driver-install1.png)
+
+    1.2. 在`版本选择`中选择希望安装的版本，点击`安装`。
+
+    1.3. 在安装界面，填写所需的安装参数，点击`确定`。
+
+    ![ofed-driver-install2](../../../images/ofed-driver-install2.png)
```
Comment (windsonsea) [inline docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md:60]:
```suggestion
    1. 进入 __Helm 应用__ -> __Helm 模板__ ，找到并点击 __ofed-driver__

        ![ofed-driver-install1](../../../images/ofed-driver-install1.png)

    2. 在 __版本选择__ 中选择希望安装的版本，点击 __安装__ 。

    3. 在安装界面，填写所需的安装参数后，点击 __确定__ 。

        ![ofed-driver-install2](../../../images/ofed-driver-install2.png)
```
UI 词用粗体比较通用

**Comment by ty-dc** [comment]
> If ready, pls remove the label `wip-do-not-merge`.

done

File: docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+    3.3. 在安装界面，通过`参数配置`选择 `YAML` 开启 driver.rdma.enabled、driver.rdma.useHostMofed 参数，让 gpu-operator 去安装 [nvidia-peermem](https://network.nvidia.com/products/GPUDirect-RDMA/) 内核模块，启用 GPUDirect RMDA 功能，加速 GPU 和 RDMA 网卡之间的转发性能，其他参考采取默认即可，然后点击`确定`，完成安装。
+
+    ![gpu-operator-install2](../../../images/gpu-operator-install2.png)
+
+    可在主机上输入如下命令，确认安装成功的内核模块填写所需的安装参数，例如：
+
+    ```bash
+    $ lsmod | grep nvidia_peermem
+    nvidia_peermem         16384  0
+    ```
```
Comment (windsonsea) [inline docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md:120]:
```suggestion
    1. 进入 __Helm 应用__ -> __Helm 模板__ ，找到并点击 gpu-operator

        ![gpu-operator-install1](../../../images/gpu-operator-install1.png)

    2. 在 __版本选择__ 中选择希望安装的版本后，点击 __安装__ 。

    3. 在安装界面，通过 __参数配置__ 选择 __YAML__ 开启 driver.rdma.enabled、driver.rdma.useHostMofed 参数，让 gpu-operator 去安装 [nvidia-peermem](https://network.nvidia.com/products/GPUDirect-RDMA/) 内核模块，启用 GPUDirect RMDA 功能，加速 GPU 和 RDMA 网卡之间的转发性能，其他用默认值，点击 __确定__ ，完成安装。

        ![gpu-operator-install2](../../../images/gpu-operator-install2.png)

        可在主机上输入如下命令，确认安装成功的内核模块填写所需的安装参数，例如：

        ```bash
        $ lsmod | grep nvidia_peermem
        nvidia_peermem         16384  0
        ```
```

File: docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+
+    1.1. 进入 `Helm 应用` -> `Helm 模板`，找到并点击 rdma-tools
+
+    ![rdma-tools-install1](../../../images/rdma-tools-install1.png)
+
+    1.2. 在安装界面，通过`参数配置`选择 `YAML`，更新配置如下：
+
+    ![rdma-tools-install2](../../../images/rdma-tools-install2.png)
+
+    可参考的 Yaml 配置如下：
```
Comment (windsonsea) [inline docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md:238]:
```suggestion
    1. 进入 __Helm 应用__ -> __Helm 模板__ ，找到并点击 rdma-tools

        ![rdma-tools-install1](../../../images/rdma-tools-install1.png)

    2. 在安装界面，通过 __参数配置__ 选择 __YAML__ ，更新配置如下：

        ![rdma-tools-install2](../../../images/rdma-tools-install2.png)

        可参考的 YAML 配置如下：
```

File: docs/zh/navigation.yml [hunk 4]
```diff
@@ -487,6 +487,8 @@ nav:
                       - Spiderpool 安装: network/modules/spiderpool/install/install.md
                       - RDMA 环境准备及安装: network/modules/spiderpool/install/rdmapara.md
                       - 安装 Nvidia OFED 驱动: network/modules/spiderpool/install/ofed_driver.md
+                      - AI 集群基于 Macvlan 技术给容器提供 RDMA 通信能力:  network/modules/spiderpool/install/ai_cluster_with_macvlan.md
```
Comment (windsonsea) [inline docs/zh/navigation.yml:490]:
```suggestion
                      - 为 AI 集群提供 RDMA 能力:  network/modules/spiderpool/install/ai_cluster_with_macvlan.md
```
左侧的 nav 空间狭窄，能缩短一些吗。正文的 title 可以很长

**Review by windsonsea — COMMENTED** [review]
2级的序号，可以让 markdown 自己生成

File: docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md (hunk 2 above)
Comment (ty-dc) [reply docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md:120]: done

File: docs/zh/navigation.yml (hunk 4 above)
Comment (ty-dc) [reply docs/zh/navigation.yml:490]: done

File: docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md (hunk 1 above)
Comment (ty-dc) [reply docs/zh/docs/network/modules/spiderpool/install/ai_cluster_with_macvlan.md]: done

**Review by windsonsea — APPROVED** [review]
The [preview](https://deploy-preview-5386--daocloud-docs.netlify.app/network/modules/spiderpool/install/ai_cluster_with_macvlan) looks good now.

**Pull request merged**